utils/wasm-gen/src/config.rs

```diff
@@ -165,7 +165,7 @@ impl<T> Default for StandardGearWasmConfigsBundle<T> {
         Self {
             log_info: Some("StandardGearWasmConfigsBundle".into()),
             existing_addresses: None,
-            remove_recursion: false,
+            remove_recursion: true,
```

Comment:
limit_recursions

Reply:
If this is set, then `gr_gas_available` must be defined in the config. The invariant must be implemented.

Reply:
I added `gr_gas_available` syscall to the config and renamed some things to `critical_gas_limit` in commit 4802c06ddacd8b0e1a0d896d62c95c060f42e0d9.